tests/pre_processing/test_extract_landscape.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import unittest
+import os.path
+
+from src.pre_processing.extract_landscape import get_facial_vectors, _extract_photos_from_file
```

Comment:
removed _exract_photo_from_file as functionality moved to separate class and equivalent tests in tests/extract_data